core/src/test/scala/scalacss/full/InlineTest.scala

```diff
@@ -242,8 +243,33 @@ object MyInlineComplexCond extends StyleSheet.Inline {
     )
 }
 
+object MyInlineWithFontFace extends StyleSheet.Inline {
+  import dsl._
+
+  val ff = fontFace("myFont", "url(font.woff)")
+    .fontStretch.expanded
+    .fontStyle.italic
+    .unicodeRange(0, 5).register
+  val ff2 = fontFace("myFont2", "url(font2.woff)")
+    .fontStyle.oblique
+    .fontWeight._200.register
+  val ff3 = fontFace("myFont3", "local(HelveticaNeue)", "url(font2.woff)")
+    .fontStretch.ultraCondensed
+    .fontWeight._200.register
+  val ff4 = fontFace("myFont3", "local(HelveticaNeue)", "url(font2.woff)").register
```

Comment:
Having `.register` at the end is inconsistent with other usage, eg: `style(...)`. How about something like this?

`def fontFace(family: String, src: String)`
`def fontFace(family: String, src: String, f: FontFace => FontFace)`
and they both return an immutable font face - no more editing or `.fontStretch` methods.

So for example:
```scala
val ff3 = fontFace("myFont3", "local(HelveticaNeue)", "url(font2.woff)",
  _.fontStretch.ultraCondensed.fontWeight._200)

val ff4 = fontFace("myFont3", "local(HelveticaNeue)", "url(font2.woff)")

ff3.fontStretch // ← doesn't work. Already registered.
ff4.fontStretch // ← doesn't work. Already registered.
```